### DISCUSSION TITLE
About ltsp nfs

### DISCUSSION THREAD

**Discussion by hc2021hc** [ltsp/ltsp#475] [Q&A] [answered]
Hello, I'm a newbie to LTSP and I'm currently setting up the LTSP server in Ubuntu 20.04.  I have set up the internal IP address of the server to be, say, 192.9.XX.XX (i.e. not the suggested 192.168.67.1).  When I run ltsp nfs, it shows 
RTNETLINK answers: Network is unreachable
LTSP command failed: ip -o route get 192.168.67.1
Could not parse output of: ip -o route get 192.168..67.1 
LTSP command failed: ip -o route get 192.168.67.2
Could not parse output of: ip -o route get 192.168..67.2
Installed /usr/share/ltsp/server/nfs/ltsp-nfs/exports in /etc/exports.d/ltsp-nfs.exports
Restarted nfs-kernel-server 

May I know what is required to be changed so that it can get my configured IP address instead?

**Comment by alkisg** [comment, marked as answer]
These commands are supposed to detect the route to the LTSP server.
Even if your internal subnet is different, a route to 192.168.67.x should still point to the "default interface".
Which means that you probably have some wrong routes declared.

What is the output of `ip -o route get 192.168.67.1`, so that we see why it fails?
Additionally, doesn't `ltsp nfs` finish successfully, even after these warnings?
Btw, why do you need a different internal subnet? That's supposed to be specific to LTSP...

**Comment by hc2021hc** [comment]
Thanks, the error is eliminated after I add a route to 192.168.67.0.
